README.md

```diff
@@ -70,10 +70,19 @@ To run the samples:
 export AWS_SECRET_ACCESS_KEY=<YourAWSSecretAccessKey>
 export AWS_ACCESS_KEY_ID=<YourAWSAccessKey>
 ```
-For audio+video, run `./kvsAacAudioVideoStreamingSample <channel-name`
-For video only, run `./kvsVideoOnlyStreamingSample <channel-name`
+For audio+video, run `./kvsAudioVideoStreamingSample <channel-name> <streaming-duration-in-seconds> <sample-location> <audio-codec>`
 
-This will stream the video/audio files from the `samples/h264SampleFrames` and `samples/aacSampleFrames` respectively.
+The last three arguments are optional. By default, 
+* the `streaming-duration-in-seconds` is 20 seconds
+* `sample-location` is `../samples`
+* `audio-codec` is `aac`
+
+If you want to use the sample for `PCM_ALAW/G.711` frames, run 
+`./kvsAudioVideoStreamingSample <channel-name> <streaming_duration> <sample_location> alaw`
+
+For video only, run `./kvsVideoOnlyStreamingSample <channel-name>`
+
+This will stream the video/audio files from the `samples/h264SampleFrames` and `samples/aacSampleFrames` respectively by default. 
```

Comment:
Missed this yest, for video only, `samples/aacSampleFrames` will not be used.

Reply:
removed `samples/aacSampleFrames` from this section and added it to the audio+video section